sp/src/game/server/mapbase/ai_grenade.h

```diff
@@ -91,6 +91,45 @@ template <class BASE_NPC>
 class CAI_GrenadeUser : public BASE_NPC, public CAI_GrenadeUserSink
 {
 	DECLARE_CLASS_NOFRIEND( CAI_GrenadeUser, BASE_NPC );
+
+	// facilitate ADL
+public:
+	using BASE_NPC::GetActiveWeapon;
```

Comment:
I would actually much rather have every occurrence to the entity be `this->` rather than using these `using`s. I've noticed Valve's own template classes use `this->`, but I never realized this is why they did that. I want to maintain consistency between Mapbase's classes and Valve's classes.

Reply:
Ah, I didn't notice that. I agree, consistency is good. I'll change it then.